lib/vector-core/src/config/global_options.rs

```diff
@@ -118,7 +118,8 @@ pub struct GlobalOptions {
 }
 
 impl GlobalOptions {
-    /// Resolve the `data_dir` option in either the global or local config, and
+    /// Resolve the `data_dir` option in either the g
+    /// lobal or local config, and
```

Comment:
🤔 this seems like an odd change. Intentional? If the line crossed a length limit boundary of some kind, I'd expect it at least to not cut off mid-word.